File: eFrizer/eFrizer.Model/HairSalonService/HairSalonService.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eFrizer.Model
{
    public class HairSalonService
    {
        public int Id { get; set; }

        public string Description { get; set; }
        public float Price { get; set; }
        public int TimeMin { get; set; }

        public int ServiceId { get; set; }
        public virtual Service Service { get; set; }
        public string ServiceName => Service?.Name;

        public int HairSalonId { get; set; }
        public virtual HairSalon HairSalon { get; set; }

        public int ImageId { get; set; }
    }
}

File: eFrizer/flutter_efrizer/lib/models/hairsalon_service/hairsalon_service.dart
class HairSalonService{
  final int hairSalonServiceId;
  final String description;
  final int price;
  final int timeMin;
  final int serviceId;
  final String serviceName;

  HairSalonService({
    required this.hairSalonServiceId,
    required this.description,
    required this.price,
    required this.serviceId,
    required this.timeMin,
    required this.serviceName
  });


  factory HairSalonService.fromJson(Map<String, dynamic> json) {
    return HairSalonService(
      hairSalonServiceId: int.parse(json['hairSalonServiceId'].toString()),
      description: json['description'],
      price: int.parse(json['price'].toString()),
      timeMin: int.parse(json['timeMin'].toString()),
      serviceId: int.parse(json['serviceId'].toString()),
      serviceName: json['serviceName']
    );
  }

}